src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
To be clear, I'm not saying that what you've done here is wrong, I'm trying to dig into it to determine:

1. If this is solved in a newer version of babel-eslint (as @zpao suggested), or
2. If this is a bug that they might like to know about

Can you check if 1 or 2 are at play?